utils.py

```diff
@@ -238,4 +230,13 @@ def get_hosts_from_env(env):
             password = env[role]["password"]
             hosts.append( (role, 0, name, host, username, password) )
 
+    # nodes
```

Comment:
why are we moving code after?

Reply:
We want the qdevice come before the nodes in the hosts array, so that later we execute qdevice salt functions before those on the nodes.

Reply:
ah ok maybe we can add a comment on this